package.json

```diff
@@ -339,13 +341,14 @@
     "@nx/nest": "16.9.0",
     "@nx/next": "16.9.0",
     "@nx/node": "16.9.0",
+    "@nx/playwright": "16.9",
     "@nx/react": "16.9.0",
     "@nx/storybook": "16.9.0",
     "@nx/web": "16.9.0",
     "@nx/webpack": "16.9.0",
     "@nx/workspace": "16.9.0",
     "@openapitools/openapi-generator-cli": "1.0.15-4.3.1",
-    "@playwright/test": "1.29",
+    "@playwright/test": "1.42.0",
```

Comment:
Maybe too big of a version jump. Alas, it's only minor versions, but a lot has changed...

Reply:
Seems to be OK